pkg/ipam/poolallocator/allocator.go

```diff
@@ -272,6 +309,63 @@ func (a *IPPoolAllocator) AllocateNext(state v1alpha2.IPAddressPhase, owner v1al
 	return ip, subnetSpec, err
 }
 
+// AllocateReserved allocates the reserved IP if exists, else allocates next
```

Comment:
if it exists

And the paragraph are not aligned.

Reply:
Fixed.